internal/csi-addons/rbd/volumegroup.go

```diff
@@ -192,6 +194,12 @@ func (vs *VolumeGroupServer) DeleteVolumeGroup(
 	// resolve the volume group
 	vg, err := mgr.GetVolumeGroupByID(ctx, req.GetVolumeGroupId())
 	if err != nil {
+		if errors.Is(err, group.ErrRBDGroupNotFound) {
+			log.DebugLog(ctx, "VolumeGroup %q doesn't exists", req.GetVolumeGroupId())
+
+			return &volumegroup.DeleteVolumeGroupResponse{}, nil
+		}
+
 		return nil, status.Errorf(
```

Comment:
Can you modify this error message? It also says could not find volume group.

Reply:
Done